src/move-node.js

```diff
@@ -17,6 +17,41 @@ const moveNode = async (toNode) => {
     const oldNodes = await moveShard.moveShard(shard, toNode);
     removedNodes.push(...oldNodes);
   }
+  return removedNodes;
+};
+
+const replaceNodeInClusteredCouch = async (toNode, shardMapJson) => {
```

Comment:
We all know this is CouchDb nodes we are replacing. Can you rename this to `replaceInCluster`?